docs/integrations/packs-format.md

```diff
@@ -38,10 +38,65 @@ will have to be filled by you. An explanation for each of them will be provided
+| serverMinVersion | String | Cortex XSOAR minimum version from which the pack is available in the format `x.x.x`, e.g. "5.5.0" |
+| currentVersion | String | The pack version, in the format of `x.x.x`. On the initial release this should be set to "1.0.0". |
+| author | String | The name of the organization / developer which developed the integration. |
+| url | String | The URL to which users should refer to in case of support needed regarding the pack. Usually is the organization support URL or the developer GitHub repository. |
+| email | String | The email address to which users should reach out to in case of support needed regarding the pack. |
+| categories | List | The use-case categories which are implemented in the pack. Usually set by the integration, which included in the pack category. Should be one of the following:<br />1. Analytics & SIEM<br />2. Utilities<br />3. Messaging<br />4. Endpoint<br />5. Network Security<br />6. Vulnerability Management<br />7. Case Management<br />8. Forensics & Malware Analysis<br />9. IT Services<br />10. Data Enrichment & Threat Intelligence<br />11. Authentication<br />12. Database<br />13. Deception<br />14. Email Gateway|
+| tags | List | Tags to be attached to the pack on Cortex XSOAR marketplace. |
+| created | String | Pack creation time in ISO 8601 format - YYYY-MM-DDTHH:mm:ssZ, e.g. 2020-01-25T10:00:00Z |
+| updated | String | Pack last update time in ISO 8601 format - YYYY-MM-DDTHH:mm:ssZ, e.g. 2020-01-25T10:00:00Z |
+| beta | Boolean | Whether the pack is in beta version or not. |
+| deprecated | Boolean | Whether the pack is deprecated or not. |
+| useCases | List | Use-cases implemented by the pack. |
+| keywords | List | List of strings by which the pack can be found in Cortex XSOAR marketplace. |
+| dependencies | Dictionary | An object that describes the content packs that the pack is dependant on. Should be kept empty on pack creation, as it is calculated by Cortex XSOAR content infrastructure. |
+| displayedImages | List | Images to be displayed in Cortex XSOAR marketplace. |
+
+Pack metadata contents for example:
+
+```json
+{
+    "name": "Palo Alto Networks Cortex XDR - Investigation and Response",
+    "description": "Cortex XDR is the world's first detection and response app that natively integrates network, endpoint and cloud data to stop sophisticated attacks.",
+    "support": "xsoar",
+    "serverMinVersion": "5.5.0",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR  ",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Endpoint"
+    ],
+    "tags": [
+        "Recommended by Cortex XSOAR",
+        "xdr"
+    ],
+    "created": "2020-03-11T13:16:53Z",
+    "updated": "2020-03-11T13:16:53Z",
+    "beta": false,
+    "deprecated": false,
+    "useCases": [
+        "Malware"
+    ],
+    "keywords": [
+        "adaptive cyber protection",
+        "apt"
+    ],
+    "dependencies": {},
```

Comment:
Lets add an example with few dependencies. This will help to understand the structure of dependencies and dependencies can be modified manually too.
The same for displayedImages

Reply:
done